# HG changeset patch
# User Francois Bissey <francois.bissey@canterbury.ac.nz>
# Date 1309486232 -43200
# Node ID d0099fa1effb838486f4e30dd58461808c99d7ba
# Parent  4ad86990d7215498658440de04560aa028dcdeeb
trac 11560 fixing doctest involving sympy. sympy 0.7.0 switched to lexicographic ordering.

diff --git a/sage/calculus/test_sympy.py b/sage/calculus/test_sympy.py
--- a/sage/calculus/test_sympy.py
+++ b/sage/calculus/test_sympy.py
@@ -152,7 +152,7 @@
     sage: type(e)
     <class 'sympy.core.add.Add'>
     sage: e
-    cos(x) + sin(y)
+    sin(y) + cos(x)
     sage: e=e._sage_()
     sage: type(e)
     <type 'sage.symbolic.expression.Expression'>
diff --git a/sage/symbolic/expression_conversions.py b/sage/symbolic/expression_conversions.py
--- a/sage/symbolic/expression_conversions.py
+++ b/sage/symbolic/expression_conversions.py
@@ -557,7 +557,7 @@
         (x, y)
         sage: f = exp(x^2) - arcsin(pi+x)/y
         sage: f._sympy_()
-        -asin(pi + x)/y + exp(x**2)
+        exp(x**2) - asin(x + pi)/y
         sage: _._sage_()
         -arcsin(pi + x)/y + e^(x^2)
 
@@ -589,7 +589,7 @@
             sage: s = SympyConverter()
             sage: f = x + 2
             sage: s.arithmetic(f, f.operator())
-            2 + x
+            x + 2
         """
         import sympy
         operator = arithmetic_operators[operator]
